src/data/concepts.json

```diff
@@ -0,0 +1,192 @@
+[{
+    "name": "iOS",
```

Comment:
This JSON is not need IMO. 
The image should come from the featuredImage in the graphQL query on the markdown pages. 
Title is also maintained on the markdown.
URL is slug of the page presumably so also in the markdown.